substrate/frame/proxy/src/lib.rs

```diff
@@ -357,11 +357,14 @@ pub mod pallet {
 			T::Currency::reserve(&who, deposit)?;
 
 			Proxies::<T>::insert(&pure, (bounded_proxies, deposit));
+			let extrinsic_index = <frame_system::Pallet<T>>::extrinsic_index().ok_or_else(|| Error::<T>::BadContext)?;
```

Comment:
```suggestion
			let extrinsic_index = <frame_system::Pallet<T>>::extrinsic_index().unwrap_or_default();
```

in `pure_account` we are doing the same.